Example/Podfile

```diff
@@ -15,13 +15,13 @@ end
 target 'ConsentViewController_ExampleTests' do
   pod 'ConsentViewController', :path => '../'
   pod 'Quick'
-  pod 'Nimble'
+  pod 'Nimble', '9.1.0'
```

Comment:
Why did you have to pin point the version to `9.1.0`?

Reply:
Please note that this branch was originally made by Vilas

Reply:
@andresilveirah fixed by commit below. Pls review